0X_LICENSE

```diff
@@ -0,0 +1,34 @@
+Licensing inclusion for the following files which were originally taken from the 0xcert ERC721 token contracts and later modified by us:
+
+contracts/0xcert/ERC165.sol
+contracts/0xcert/SupportsInterface.sol
+
+test/ERC165/SupportsInterface.test.js
+test/ERC165/behaviors/ERC165.behavior.js
+
+
+
+--
+
+The MIT License (MIT)
+
+Copyright (c) 2018 0xcert admin@0xcert.org.
+
+Permission is hereby granted, free of charge, to any person obtaining
+a copy of this software and associated documentation files (the
+"Software"), to deal in the Software without restriction, including
+without limitation the rights to use, copy, modify, merge, publish,
+distribute, sublicense, and/or sell copies of the Software, and to
+permit persons to whom the Software is furnished to do so, subject to
+the following conditions:
+
+The above copyright notice and this permission notice shall be included
+in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
```

Comment:
Create a new directory where we can stash all these external licenses instead of having them in root